internal/ui/oplog/operation_log.go

```diff
@@ -59,6 +81,8 @@ func (m *Model) ShortHelp() []key.Binding {
 	return []key.Binding{
 		m.keymap.Up,
 		m.keymap.Down,
+		m.keymap.ScrollUp,
+		m.keymap.ScrollDown,
```

Comment:
These scroll up/down clean up is good. I am happy to merge those if you can extract those into a separate PR.

Reply:
drafted another PR #437, have a look!